_sass/_scaffolding.scss

```diff
@@ -1,16 +1,22 @@
+// TODO: suggest adding box-sizing: border-box; to all
+// * {
+//   box-sizing: border-box;
+// }
+
```

Comment:
I would recommend doing this! I sent you a message on slack about it, it has to do with how folks typically think about the size of an element. It's a change that would need a fair amount of QA across the site to make sure there aren't any shenanigans, but it's something I would recommend doing!